test/instrumentation/modules/redis4-legacy.test.js

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright Elasticsearch B.V. and other contributors where applicable.
+ * Licensed under the BSD 2-Clause License; you may not use this file except in
+ * compliance with the BSD 2-Clause License.
+ */
+
```

Comment:
Please add a comment here that this test file is about testing `legacyMode`. Perhaps link to https://github.com/redis/node-redis/blob/HEAD/docs/v3-to-v4.md#legacy-mode.  I think it could be confusing to a future reader that this isn't about testing a legacy version of the module.

Reply:
ceb85722350e61d82de004cdf82735a0169d288e d7bb6e03fa97c1f11b29a5a0a9449d7aec9ac1e7